demo/CMakeLists.txt

```diff
@@ -0,0 +1,87 @@
+# Copyright 2022 The Chromium Authors.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+cmake_minimum_required(VERSION 3.22)
+
+project(chrome_enterprise_connector_local_analysis)
+
+# Makes available the package definitions in vcpkg.
+include(build/vcpkg/scripts/buildsystems/vcpkg.cmake)
+
+find_package(Protobuf CONFIG REQUIRED)
+
+# The include directory contains the header files needed by the demo code.
+# The gen directory contains generated protobuf headers describing the request
+# and response objects used to communicate with Google Chrome.
+include_directories(
+  "${PROJECT_SOURCE_DIR}/../agent/include"
+  "${PROJECT_SOURCE_DIR}/../browser/include"
+  "${PROJECT_BINARY_DIR}/gen"
+)
```

Comment:
Let's not mix the agent include path with the client include path.  To specify different include paths for agent and client, use [target_include_directories](https://cmake.org/cmake/help/git-master/command/target_include_directories.html).

Maybe remove the `include_directories` command and define two variables:
```
set(AGENT_INCLUDE_PATH
  "${PROJECT_SOURCE_DIR}/../agent/include"
  "${PROJECT_BINARY_DIR}/gen"
)

set(CLIENT_INCLUDE_PATH
  "${PROJECT_SOURCE_DIR}/../browser/include"
  "${PROJECT_BINARY_DIR}/gen"
)
```
then add the appropriate `target_include_directories(<target> <TARGET>_INCLUDE_PATH)` lines under the corresponding `add_library()` and `add_executable()` commands.

Reply:
Done.